src/main/java/uk/gov/hmcts/reform/profilesync/service/impl/ProfileUpdateServiceImpl.java

```diff
@@ -74,21 +80,20 @@ public void updateUserProfile(String searchQuery, String bearerToken, String s2s
     private void syncUser(String bearerToken, String s2sToken,
                           String userId, UserProfile updatedUserProfile) throws UserProfileSyncException {
 
-        log.info("Inside  syncUser:: method");
+        log.info(loggingComponentName, "Inside  syncUser:: method");
         Response response = userProfileClient.syncUserStatus(bearerToken, s2sToken, userId, updatedUserProfile);
 
-        log.info("Body response::" + response.body().toString());
+        log.info(loggingComponentName, "Body response::" + response.body().toString());
         if (response.status() > 300) {
 
-            log.error("Exception occurred while updating the user profile: Status - {}" + userId + ":"
-                    + updatedUserProfile.getIdamStatus());
-            log.error("Exception occurred while updating the user profile: http Status - {}", response.status());
+            log.error(loggingComponentName, "Exception occurred while updating the user profile: Status - {}"
+                    + userId + ":" + updatedUserProfile.getIdamStatus());
```

Comment:
can you please remove userId

Reply:
Addressed in this commit: https://github.com/hmcts/rd-profile-sync/pull/293/commits/73ac6e520f779dcba475bf1df0cdb225cbf5b222